### PR TITLE
Add Redis support

### PR DESCRIPTION
**Pull request by RobLoach**
https://github.com/RobLoach/drush-docker/issues/22

### CODE REVIEW

**Comment by pirog** [comment]
@RobLoach hoping to have a chance to test this a little and add in the imagickmagix stuffs as well.

That said, i've added some inline questionz.

File: base/Dockerfile [hunk 1]
```diff
@@ -6,5 +6,14 @@ MAINTAINER Rob Loach <robloach@gmail.com>
 RUN apt-get update && apt-get install -y mysql-client
 RUN docker-php-ext-install pdo_mysql
 
+# Add the Redis PHP module.
+RUN git clone --branch="php7" https://github.com/phpredis/phpredis.git /tmp/phpredis && cd /tmp/phpredis && \
```
Comment (pirog) [inline base/Dockerfile]: @RobLoach is there a particular reason why we aren't using the built in `docker-php-ext-install`? I *think* that is the preferred way to do things

```bash
git clone -b php7 https://github.com/phpredis/phpredis.git /usr/src/php/ext/redis
docker-php-ext-install redis
```

See: http://stackoverflow.com/questions/31369867/how-to-install-php-redis-extension-using-the-official-php-docker-image-approach

Comment (RobLoach) [inline base/Dockerfile]: Oh I like that.

**Comment by pirog** [comment]
@RobLoach 

💯 x RIKER = 
image

**Pull request merged**